package.json

```
@@ -26,26 +26,27 @@
   "main": "lib/redux.js",
   "unpkg": "dist/redux.js",
   "module": "es/redux.js",
-  "typings": "./index.d.ts",
+  "typings": "types/index.d.ts",
```

```suggestion
  "types": "types/index.d.ts",
```

Just pointing out that "types" is [what you see most places and is synonymous with "typings"](https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#including-declarations-in-your-npm-package).

---

Yes, but `types` is the primary name. Other tools are more likely to implement that than `typings` (it's easy to miss the alias).

---

I agree. `types` is the primary name; hence, my suggestion to change it to "types" above.

---

Sorry, I was reading your suggestion as the actual source. Whoops!